portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,21 @@ function addRandomFunFact() {
   const factContainer = document.getElementById('fact-container');
   factContainer.innerText = fact;
 }
+
+function getRandomGreeting() {
+    const greetings =
+      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+
+    // Pick a random greeting.
+    var greeting = greetings[Math.floor(Math.random() * greetings.length)];
+    return greeting;
+}
+
+/**
+ * Fetches data from server and adds it to home page.
```

Comment:
Not an actionable comment but wanted to call out that this is a great example of a method comment, having comments like these throughout the code can be really helpful as your code expands, and especially as you start collaborating with other engineers on the same code :)